src/embed-images.ts

```diff
@@ -53,9 +53,9 @@ async function embedImageNode<T extends HTMLElement | SVGImageElement>(
   const url = isImageElement ? clonedNode.src : clonedNode.href.baseVal
 
   const dataURL = await resourceToDataURL(url, getMimeType(url), options)
-  await new Promise((resolve, reject) => {
+  await new Promise((resolve) => {
     clonedNode.onload = resolve
-    clonedNode.onerror = reject
+    clonedNode.onerror = resolve
```

Comment:
With the changes we've made, an error can occur ?

Reply:
I think so: image not found, CORS, ...